publish2cloud.py

```diff
@@ -265,11 +265,11 @@ def publish_to_cloud(config, chunknum, check_versioning=None):
                 continue
 
             version = p_version.parse(config.get(section, 'version'))
-            skip_sv_separation = (
-                version.release[0] < VER_SV_SEPARATION_STARTED
+            skip_large_entity_separation = (
+                version.release[0] < VER_LARGE_ENTITIES_SEPARATION_STARTED
                 and section in LARGE_ENTITIES_SECTIONS
```

Comment:
same comment as above - do we need this `and` clause here?

Reply:
As responded in `lists2safebrowsing.py` 
> Yes, because we do want versioned standard entity lists if the version is lower than the current Nightly(74) but we want `skip_large_entity_separation` to return `True` and exit early from this function when it tries to version large entities like Google where separation is not needed.

if we don't add this `and` it tries to upload a versioned large entity list when there is none and result to error.